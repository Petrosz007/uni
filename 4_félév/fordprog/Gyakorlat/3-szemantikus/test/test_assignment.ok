program test_assignment
    natural a
    natural b
    boolean c
    boolean d
begin
    a := 1
    b := 2
    c := false
    d := true
    a := a + b
    b := b - 1
    c := a > b or c
    d := not d
    write(a)
    write(b)
    write(c)
    write(d)
end
