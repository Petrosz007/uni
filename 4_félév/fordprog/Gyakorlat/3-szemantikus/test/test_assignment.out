3
1
true
false
